project/Common.scala

```diff
@@ -62,14 +62,16 @@ object Common extends AutoPlugin {
           if (isJdk8 || scalaVersion.value == Dependencies.Scala212) Seq.empty
           else Seq("--release", "8")
         ),
+      doc / javacOptions := Nil,
```

Comment:
Not sure why this was needed, failing on my machine but not CI, I don't think we need that -X:lint when compiling apidocs anwyay.